### PR TITLE
193 edit devices

### PR DESCRIPTION
**Pull request by domi1504**

### CODE REVIEW

**Review by NicoStellwag — APPROVED** [review]
The code makes sense to me and everything works as expected, so I approve.

A small proposal, though:
The renaming is really unintuitive.
At first I clicked the rename button a couple of times and wondered why no dialogue etc. would pop up, only then I realized that the name above the button was editable.
Either make the rename button present some popup (which I would prefer), or at least make it obvious to the user that the name is a text field (by drawing a box around it and showing the cursor or sth like that).

**Comment by Chippelius** [comment]
I agree with Nico and his proposals and want to propose another solution: Get rid of the rename button entirely and make the name change every time the text field content changes (have a look at the settings page & view model for how to do that).

**Comment by domi1504** [comment]
Fair enough, thanks for the suggestions. We see your point.
Since as we've understood the UI will be redesigned anyways, we decided to apply some minor changes to the existing version, such that it is more clear to the user.
What do you think?

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemDetailPage.xaml [hunk 1]
```diff
@@ -17,7 +17,24 @@
...
+        <Button
+            x:Name="RenameButton"
+            Text="Rename"
+            BackgroundColor="White"
+            TextColor="Black"
+            BorderColor="Black"
+            BorderWidth="1"
+            Margin="3"
+            Command="{Binding RenameButtonTapped}"
+            IsVisible="{Binding UserLabelEdited}"
```
Comment (Chippelius) [inline Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemDetailPage.xaml:36]:
```suggestion
            IsEnabled="{Binding UserLabelEdited}"
```
How about this?

**Review by Interperle — APPROVED** [review]
Also works on iOS

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemDetailPage.xaml (hunk 1 above)
Comment (NicoStellwag) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemDetailPage.xaml:36]: new ux looks fine to me, honestly no strong opinion regarding disabled / not shown button from my side

Comment (Chippelius) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemDetailPage.xaml:36]: It's a proposal, but the decision is yours

**Pull request merged**